### PR TITLE
Create kubeconfig secret

### PR DESCRIPTION
**Pull request by clyang82**

### CODE REVIEW

File: controlplane/hack/deploy/kustomization.yaml [hunk 1]
```diff
@@ -44,8 +41,9 @@ secretGenerator:
   - cert/client-ca.key
   - cert/serving-kube-apiserver.crt
   - cert/serving-kube-apiserver.key
-  - cert/client-kube-aggregator.crt
-  - cert/client-kube-aggregator.key
   - cert/kube-aggregator.kubeconfig
+- name: controlplane-kubeconfig
```
Comment (qiujian16) [inline controlplane/hack/deploy/kustomization.yaml:45]: this may have issue when generating cluster-info cm?

Comment (ycyaoxdu) [reply controlplane/hack/deploy/kustomization.yaml:45]: looks like the cert/kube-aggregator.kubeconfig stiil exists in secret controlplane-cert @qiujian16

Comment (clyang82) [reply controlplane/hack/deploy/kustomization.yaml:45]: yes. cert/kube-aggregator.kubeconfig is still using for generating cluster-info cm.

Comment (qiujian16) [reply controlplane/hack/deploy/kustomization.yaml:45]: but this one is not useful because it has no related cert/key, right?

**Pull request merged**